src/I18n/Formatter/IcuFormatter.php

```diff
@@ -51,7 +51,7 @@ public function format($locale, $message, array $vars)
             return $this->_formatMessage($locale, $message, $vars);
         }
 
-        if (isset($vars['_context']) && isset($message['_context'])) {
+        if (isset($vars['_context'], $message['_context'])) {
```

Comment:
Yes, but it is hidden in the documentation, whereas the && makes this very clear in the code itself.
And to be honest, it doesn't safe that much of code.